.github/workflows/rav1e.yml

```diff
@@ -94,6 +97,11 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Set env vars
```

Comment:
Can we use this approach

```yaml
    - name: Set no-asm-tests environment variables
      if: matrix.conf == 'no-asm-tests'
      run: |
        echo "::set-env name=RAV1E_CPU_TARGET::rust"
```

instead of an external action?

Reply:
Sure, I just used the recommendation from the Github Actions docs. This is honestly better.